config.tutorial.yaml

```diff
@@ -337,8 +337,8 @@ monte_carlo:
       # Key: add below the pypsa object for the monte_carlo sampling, "network" is only allowed for filtering!
       # Value: currently supported format [l_bound, u_bound] or empty [], represent multiplication factors for the object
     loads_t.p_set: [0.9, 1.1]
-    generators_t.p_max_pu.loc[:, n.generators.carrier == "wind"]: [0.9, 1.1]
-    generators_t.p_max_pu.loc[:, n.generators.carrier == "solar"]: [0.9, 1.1]
+    # generators_t.p_max_pu.loc[:, n.generators.carrier == "wind"]: [0.9, 1.1]
```

Comment:
@davide-f we need at least two rows to test the Monte-Carlo stuff. Test is currently failing because of that

Reply:
added a commit... let's see

Reply:
I recreated the PR and kept this line commented out.
pretty YAML crashes with checking those lines. In this PR, there is a fix to monte_carlo caospy to make sure that it works even with a single parameter in the list of monte carlos. The other options have been tested as well.